cmd/backup/exec.go

```diff
@@ -51,19 +51,15 @@ func (s *script) exec(containerRef string, command string, user string) ([]byte,
 		outputDone <- err
 	}()
 
-	select {
```

Comment:
Not quite sure what the purpose of the select is here.
Because there is only one case, it will block/wait anyway -> remove the select and just reading straight from the channel.
This also means, that the go func right over this doesn't do anything as we wait here anyway.
Or do I miss something?

Reply:
If I recall correctly, most of this code is copied verbatim from the test suite of `moby` (see the license header). I don't know too much about it, and I'd maybe just leave it unchanged?

Your reasonsing sounds correct though.